docs/architecture/README.md

```diff
@@ -0,0 +1,36 @@
+# Architecture Decision Records (ADR)
+
+This is a location to record all high-level architecture decisions in the tendermint project.
```

Comment:
This was probably a copy-pasta, but should be updated to be explorer specific.

Reply:
😞 Here is a pr to fix that issue: https://github.com/provenance-io/explorer-service/pull/171/files @crussell-figure